data/json/items/melee/swords_and_blades.json

```
@@ -158,7 +158,7 @@
   {
     "id": "knife_hunting",
     "type": "TOOL",
-    "category": "tools",
+    "category": "tools_farming",
```

All other knifes are `tools_workshop` it seems so uh...
```suggestion
    "category": "tools_workshop",
```